tests/integration/test_integration.py

```diff
@@ -23,10 +29,57 @@ async def charm(self, ops_test):
         return charm
 
     @staticmethod
-    def get_certificate_from_file(filename: str) -> str:
-        with open(filename, "r") as file:
-            certificate = file.read()
-        return certificate
+    def get_certificate_from_csr(csr: str) -> dict:
```

Comment:
Does it make sense to add this function to `test_helpers.py` which also has the `get_certificate_from_file`, and make that file common for both unit and integration tests?

Reply:
Oh I see we are not using `get_certificate_from_file` anymore :)
Then it's not important, unless this function would be used in unit tests too.

Reply:
It is only used on the integration tests. At the moment we don't need to use this function in the UTs since they are hard coded and valid